src/test/java/io/neonbee/endpoint/odatav4/internal/olingo/processor/ProcessorHelperTest.java

```diff
@@ -36,11 +37,12 @@ void transferResponseHint() {
         dataContext.responseData().put(ODATA_FILTER_KEY, Boolean.TRUE);
         dataContext.responseData().put(ODATA_EXPAND_KEY, Boolean.FALSE);
         dataContext.responseData().put(ODATA_COUNT_SIZE_KEY, 42);
-        ProcessorHelper.transferResponseHint(dataContext, routingContext);
+        EntityWrapper result = ProcessorHelper.transferResponseHint(dataContext, routingContext, null);
```

Comment:
I'd like to use a real object for EntityWrapper, even it is a mock. But this test would ensure that transferResponseHint returns the EntityWrapper you pass into it.

Reply:
null is replaced by a mock object